lib/kitchen/transport/winrm.rb

```diff
@@ -93,14 +104,14 @@ def execute(command)
             log_stderr_on_warn(stderr)
           elsif exit_code != 0
             log_stderr_on_warn(stderr)
-            raise Transport::WinrmFailed,
-              "WinRM exited (#{exit_code}) for command: [#{command}]"
+            fail Transport::WinrmFailed,
```

Comment:
I don't remember, I think rubocop made that change or suggested it.. I'll revert it.